rust-server/src/main.rs

```diff
@@ -33,13 +32,38 @@ struct Args {
     #[arg(long)]
     #[clap(value_enum, default_value_t=Storage::JsonFiles)]
     storage: Storage,
+
+    /// Path to the directory where the database file(s) will be created
+    #[arg(long)]
```

Comment:
As above I wonder if this can be done so that the arguments that only apply to postgres are done separately.